blocks/library/cover-image/index.js

```diff
@@ -210,17 +211,16 @@ export const settings = {
 						inlineToolbar
 					/>
 				) : null }
-			</section>,
+			</div>,
 		];
 	},
 
 	save( { attributes, className } ) {
 		const { url, title, hasParallax, dimRatio, align, contentAlign } = attributes;
-		const style = url ?
-			{ backgroundImage: `url(${ url })` } :
-			undefined;
+		const style = backgroundImageStyles( url );
 		const classes = classnames(
 			className,
+			align ? `align${ align }` : null,
```

Comment:
This seems like a duplicate?

Reply:
I removed duplicate and fixed merge conflict.